api/src/main/java/io/minio/GetPresignedObjectUrlArgs.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * MinIO Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio;
+
+import io.minio.http.Method;
+import java.util.concurrent.TimeUnit;
+
+/** Argument class of MinioClient.getPresignedObjectUrl(). */
+public class GetPresignedObjectUrlArgs extends ObjectArgs {
+
+  // default expiration for a presigned URL is 7 days in seconds
+  public static final int DEFAULT_EXPIRY_TIME = 7 * 24 * 3600;
```

Comment:
1. Remove above new line and add new line here to differentiate the constant.
2. Please use `TimeUnit.DAYS.toSeconds(7)` for readability.